modules/nodes-nodes-working-master-schedulable.adoc

```diff
@@ -7,13 +7,14 @@
 
 As of {product-title} 4.2, you can configure master nodes to be
```

Comment:
Not in scope for this PR but wondering if you can remove "As of 4.2" now that we support only back to 4.3?

Reply:
I thought about removing that. Wanted to see if a reviewer had the same idea. Thanks!!